view/templates/content/image_grid.tpl

```diff
@@ -0,0 +1,14 @@
+<link rel="stylesheet" href="view/theme/frio/css/image_grid.css" type="text/css" media="screen"/>
+
+<div id="row" class="row">
```

Comment:
Please remove the `id` as it's likely it will be used several times in a single page, which breaks its semantics (only one unique `id` value per page).

Reply:
wilco